docs/upgrade-guide.md

```diff
@@ -2,6 +2,12 @@
 
 ## Upgrading from deck.gl v8.4 to v8.5
 
+### Transpilation
+
+The module entry point is now only lightly transpiled for the most commonly used evergreen browsers. This change offers significant savings on bundle size. If your application needs to support older browsers such as IE 11, make sure to include `node_modules` in your babel config.
```

Comment:
This section probably needs a pass. https://deck.gl/docs/developer-guide/building-apps

There used to be a similar page in luma.gl but not sure after the total website rewrite in 8.0.

There is also a blurb on loaders.gl front page on IE11 support that probably should mention this. Forgot about that when looking at that PR.